permissions/plugin-assertthat-bdd-jira.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "assertthat-bdd-jira"
+github: "jenkinsci/assertthat-bdd-jira-plugin"
+paths:
+- "org/jenkins-ci/plugins/assertthat-bdd-jira"
```

Comment:
This needs to match the groupId/artifactId from your pom.xml:

https://github.com/jenkinsci/assertthat-bdd-jira-plugin/blob/master/pom.xml#L10

So, it should be "io/jenkins/plugins/assertthat-bdd-jira"